packages/fluent-theme/src/components/suggestedActions/SuggestedAction.tsx

```diff
@@ -50,21 +49,20 @@ const styles = {
         color: 'var(--webchat-colorBrandForeground2Hover)'
       }
     },
+
     '&:not([aria-disabled="true"]):active': {
       backgroundColor: 'var(--webchat-colorBrandBackground2Pressed)',
       color: 'var(--webchat-colorBrandForeground2Pressed)'
     },
+
     '&[aria-disabled="true"]': {
-      color: ' var(--webchat-colorNeutralForegroundDisabled)',
+      color: 'var(--webchat-colorNeutralForegroundDisabled)',
       cursor: 'not-allowed'
     }
   },
 
   'webchat-fluent__suggested-action__image': {
-    width: '1em',
-    height: '1em',
-    fontSize: '20px',
-    translate: '0 -1px'
+    height: '12px'
```

Comment:
This now relies on the `width` set either by user agent or any other user-defined rule e.g. `img { width: /* some value here */ }`

While this works, this may easily break as we don't provide any css-reset for webchat

Reply:
Let's fix it in the next PR. :)